### PR TITLE
Improve create nft collections

### PR DESCRIPTION
**Pull request by Lbqds**
* Deploy NFT collections with `CopyCreateContract` instr
* Check the `maxBatchMintSize` parameter in the deployment script

### CODE REVIEW

File: utils/nft-collection.ts [hunk 1]
```diff
@@ -157,4 +166,13 @@ export class NFTCollection extends DeployHelpers {
       }
     )
   }
+}
+
+async function calcContractId(txId: string, groupIndex: number) {
```
Comment (h0ngcha0) [inline utils/nft-collection.ts:171]: Maybe not part of this PR, but feels like this could be moved to SDK

**Review by h0ngcha0 — APPROVED** [review]
LGTM :+1:

File: utils/nft-collection.ts (hunk 1 above)
Comment (Lbqds) [reply utils/nft-collection.ts:171]: Sounds great 👍, will move it to SDK.
